src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -1233,6 +1260,50 @@ private void UpdateEditor(float delta)
         {
             CurrentOrganelleCost = 0;
         }
+
+        // Apply camera movement
+        camera.ObjectToFollow.Translation += new Vector3(cameraMovement.x, 0, cameraMovement.y) * delta;
+    }
+
+    private void HandleMovement()
+    {
+        cameraMovement = Vector2.Zero;
+        if (mousePanningStart == null)
+        {
+            if (Input.IsActionPressed("e_pan_left"))
+            {
+                cameraMovement += Vector2.Left;
+            }
+
+            if (Input.IsActionPressed("e_pan_right"))
+            {
+                cameraMovement += Vector2.Right;
+            }
+
+            if (Input.IsActionPressed("e_pan_down"))
+            {
+                cameraMovement += Vector2.Down;
+            }
+
+            if (Input.IsActionPressed("e_pan_up"))
+            {
+                cameraMovement += Vector2.Up;
+            }
```

Comment:
I don't think this works correctly. You should just make the code similar to what
 PlayerMicrobeInput does so that I can sleep well knowing that the movement input handling is uniform.

Just reread and noticed this is not true:
(This is exactly the type of code that can get stuck if window focus is lost just at the right time.)

Reply:
Well, the code cannot get stuck, because I set cameraMovement to Zero in each iteration and four bools can get stuck in the same way, or am I missing something?

Reply:
I revised my comment that it can't get stuck. But the bools can get unstuck more reliably.
Because the player can press the key again that didn't get registered as unpressed.
Functionality I can't see a problem, but I really would like things to be uniform in the code base. So could you please make this use the same approach as the the player movement input does?